### ISSUE TITLE
PermissionError when tryting to start container

### ISSUE THREAD

**Issue by Doctor-Who** [Chorss/docker-pgAdmin4#13]
Hi,

OS : openSUSE Leap 64 bits
Docker Version : 17.04.0-ce
CPU | 56
Memory | 135.1 GB

When trying to start Docker with command like quickstart, 
`docker run -d -p 8000:5050 -v /srv/data/pgadmin4_data:/data chorss/docker-pgadmin4`

starting failed, see log : 
```
usermod: no changes
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/site-packages/pgadmin4/pgAdmin4.py", line 55, in <module>
    exec(open(file_quote(setupfile), 'r').read())
  File "<string>", line 46, in <module>
  File "/usr/local/lib/python3.6/site-packages/pgadmin4/pgadmin/setup/data_directory.py", line 26, in create_app_data_directory
    _create_directory_if_not_exists(os.path.dirname(config.LOG_FILE))
  File "/usr/local/lib/python3.6/site-packages/pgadmin4/pgadmin/setup/data_directory.py", line 15, in _create_directory_if_not_exists
    os.mkdir(_path)
PermissionError: [Errno 13] Permission denied: '/data/logs'
```
even with specify when I specify default OS user and associated group : 

`docker run -d -p 8000:5050 -e UID=1000 -e GID=100 -v /srv/data/pgadmin4_data:/data chorss/docker-pgadmin4`

```
groupmod: GID '100' already exists
usermod: no changes
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/site-packages/pgadmin4/pgAdmin4.py", line 55, in <module>
    exec(open(file_quote(setupfile), 'r').read())
  File "<string>", line 46, in <module>
  File "/usr/local/lib/python3.6/site-packages/pgadmin4/pgadmin/setup/data_directory.py", line 26, in create_app_data_directory
    _create_directory_if_not_exists(os.path.dirname(config.LOG_FILE))
  File "/usr/local/lib/python3.6/site-packages/pgadmin4/pgadmin/setup/data_directory.py", line 15, in _create_directory_if_not_exists
    os.mkdir(_path)
PermissionError: [Errno 13] Permission denied: '/data/logs'
```
I'm quite new about Docker but it seems I've done right but on the same machine, i!'m running other docker such as portainer, postgis, etc.

Thanks for help ;)

**Comment by marcioBigolin** [comment]
This issue  was fixed #2, but, I have problem too.

**Comment by Chorss** [comment]
Hi,

This is bug, because if it does not exist folder "pgadmin4_data", it will be created **on root's rights**
The folder was created by docker. 

**Recommend create this folder before creating a container**
